### DISCUSSION TITLE
v0.10.17 does not support databases 0.5.0

### DISCUSSION THREAD

**Discussion by HarrySky** [collerek/ormar#327] [General]
Release notes for 0.10.17 state:

> Add support for sqlalchemy >=1.4 (requires databases >= 0.5.0)

But actually in setup.py we see that databases 0.5.0 is not supported:

https://github.com/collerek/ormar/blob/47f2d61786e490efca9e15c0126fb3f598c03965/setup.py#L66

Can you please release version with databases 0.5.0 support if it is indeed supported? :smile:

**Comment by collerek** [comment]
Opps! 💩 happens!

Thanks for pointing it out :)

**Comment by collerek** [comment]
Fixed in 0.10.18 (#329)
